pkg/controller/node/nodecontroller.go

```diff
@@ -505,7 +505,7 @@ func (nc *NodeController) onNodeDelete(originalObj interface{}) {
 			glog.Errorf("Received unexpected object: %v", obj)
 			return
 		}
-		node, ok = deletedState.Obj.(*v1.Node)
+		_, ok = deletedState.Obj.(*v1.Node)
```

Comment:
@wojtek-t The quesion is that if para. node is be used later. If used, I am wrong and I will close this PR, if not used, the para. node is redundant. Do you think so?